src/main/java/com/redhat/cloud/policies/app/health/ProcSelfStatusExporter.java

```diff
@@ -47,15 +48,31 @@
 
     private boolean hasWarned = false;
 
-    long vmHwm;
-    long vmRss;
-    long rssAnon;
-    long rssFile;
-    long vmStk;
-    long vmLib;
-    long vmData;
-    long vmSize;
-    int threads;
+    @Inject
+    MeterRegistry registry;
+
+    Long vmHwm;
+    Long vmRss;
+    Long rssAnon;
+    Long rssFile;
+    Long vmStk;
+    Long vmLib;
+    Long vmData;
+    Long vmSize;
+    Integer threads;
```

Comment:
I remember seeing `Atomic*` variable in the Micrometer doc. Do we need that here?

Reply:
It actually was Atomic* before. Not sure if we need it here. Could be.. but I would need to test it with more than one pod to see if it's necessary or not.